### PR TITLE
setup associations

### PR DESCRIPTION
**Pull request by xtrahuman**
# setup associations
- created class Classroom, Book, Rental
- associated Classroom and Student
- associated Person, Rental, and Book

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rental
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+end
```
Comment (hiromataba) [inline book.rb]: - [ ] For your `Book` class you should have a method for adding rentals, the book itself should be sent as a parameter to create the new rental achieving the `has-many` association
Your code should look something like this :point_down: 

```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```

File: person.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 require './corrector'
 class Person
-  attr_reader :id
+  attr_reader :id, :rental
   attr_accessor :name, :age
 
   def initialize(age, name = 'unknown', parent_permission: true)
```
Comment (hiromataba) [inline person.rb:6]: - [ ] You should have a  method for adding rentals  for your `Person` class, the person itself should be sent as a parameter to create the new rental achieving the `has-many` association`.
Your code should look something like this :point_down: 

```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
  end
```

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi :wave: @xtrahuman  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
### To highlight :1st_place_medal: 
- No linter errors :heavy_check_mark: 
- Descriptive  README file :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please add more description for your project to make your README look more professional 

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by er-contreras — APPROVED** [review]
Hi @xtrahuman , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- I see that you did most of the work, well done!

Felix

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**